src/pages/Post/Reply/Reply.jsx

```diff
@@ -0,0 +1,38 @@
+import React from "react";
+import "./reply.css";
+import { Reply } from "@mui/icons-material";
+import { Box, Button, Avatar, TextField } from "@mui/material";
+
+const ReplyComponent = () => {
+
+    return (
+        <Box className="reply-replybox">
+            <Box className="reply-avatar">
+                <Avatar alt="name" src="https://cdn.iconscout.com/icon/free/png-256/avatar-370-456322.png" />
+            </Box>
+
+            <TextField 
+            
+                fullWidth
+                id="outlined-textarea"
+                label="Comment..."
+                placeholder=""
+                multiline
+            />
+            <Box className="reply-button">
+                <Button variant="outlined" startIcon={<Reply />}>
+                    Comment
+                </Button>
+            </Box>
+
+        </Box>
+
+
+
+
+    );
+
+
```

Comment:
Quite a lot of unnecessary empty lines...? or perhaps there's a hidden purpose. If it's a purposeless empty lines then it's better for code readability to remove them.

Reply:
Got rid of them :)